openfisca_aotearoa/parameters/benefits/rates_rebates/income_threshold.yaml

```diff
@@ -1,49 +1,49 @@
 description: Income threshold
 reference: Obtained from spreadsheet at Department Of Internal Affairs Innovation Lab
 values:
-  1973-06-01:
+  1973-07-01:
```

Comment:
Question, why is `1973-07-01` a better choice than `1973-07`?

Reply:
Just because it's explicit 😃 Whenever we model legislation, we want to give as much information written down as we could get. It would functionally be equivalent, but here we're stating that the spreadsheet we've sourced the data from explicitly stated a date of effect of the 1st of July, not just “starting in July”.